packages/kit/src/components/AddressTypeSelector/AddressTypeSelectorTrigger.tsx

```
@@ -0,0 +1,58 @@
+function AddressTypeSelectorTrigger(props: IProps) {
+  const { activeDeriveInfo, disableSelector } = props;
+  const intl = useIntl();
+
+  if (!activeDeriveInfo) {
+    return null;
+  }
+
+  return (
+    <Badge
+      userSelect="none"
+      role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)